driver-core/src/main/java/com/datastax/driver/core/Host.java

```diff
@@ -221,6 +241,32 @@ public VersionNumber getCassandraVersion() {
     }
 
     /**
+     * The DSE version the host is running.
+     * <p/>
+     * It is also possible for this information to be unavailable. In that case
+     * this method returns {@code null}, and the caller should always be aware of this
+     * possibility.
+     *
+     * @return the DSE version the host is running.
+     */
+    public VersionNumber getDseVersion() {
```

Comment:
I've noticed that this can happen even when using DSE as the information isn't populated immediately as the host becomes available on 9042, but eventually the data is populated.  This could mean the driver may also not reliably have this data even if using DSE, but doesn't seem likely (except maybe in bootstrap cases).  Do we have a way to force refresh of this data?  That might be nice to have (this would be a separate request).